test/powershell/Modules/Microsoft.PowerShell.Core/PSSessionConfiguration.Tests.ps1

```diff
@@ -1,12 +1,16 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+
+Import-Module HelpersCommon
```

Comment:
Doesn't autoloading work?

Reply:
Only if you have the test modules in your PSModulePath.  If you run the tests directly instead of via Invoke-PSPester, I figured not finding the module was a better error message than not finding the cmdlet

Reply:
I wonder that we add this again because previously we removed all explicit `import-module` from all test files. Calling Invoke-PSPester once in a session could resolves the problem.

Reply:
I'm ok either way on this.  Some of test scripts already explicitly import helper modules.